policies/policy-change-process.md

```diff
@@ -66,3 +68,17 @@ and closing the pull request without merging.
 
 If the policy change is approved, the pull request is merged to the
 master branch of the technical-policies repository.
+
+Exceptions
+----------
+
+Minor policy edits that do not change the existing meaning of the edited
+policies do not require this voting process. Typical examples of such edits
+are spelling, grammar, and formatting fixes.
+
+These edits are done via pull requests that are approved by two OTC members
+where neither of them is the author of the submission. The pull request
+should be labelled with the `minor edit` label.
```

Comment:
Should both members have to explicitly agree to that the normal process does not need to be followed. Or is the label and 24 hour timer sufficient?

Reply:
I think the latter.  We do this for branches on the main repo, why not here too?
CLA: trivial does it the opposite way around...

Reply:
Didn't we decide last week? to change `should be labelled` to `must be labelled`?

But never mind, that's no show-stopper.